### ISSUE TITLE
Incorrect lambda indentation

### ISSUE THREAD

**Issue by joca-bt** [checkstyle/checkstyle#7809]
We're using double indentation for wrapping function calls, however lambdas seem to have some issue with it.

```
        return dsl.select(...)
            .from(...)
            .where(...)
            .fetch()
            .stream()
            .filter(...)
            .collect(
                    Collectors.groupingBy(
                            record -> converter.apply(record.get(getItemField())),
                            Collectors.toMap(...)));
```
Flags
```
[WARNING] <file>:[<line>] (indentation) Indentation: 'lambda arguments' has incorrect indentation level 28, expected level should be 24.
```
This only happens with lambdas.

Using the default configuration, ie. `<module name="Indentation"/>`.

**Comment by romani** [comment]
Please strictly follow issy template to give all required details.

**Comment by Abhishek-kumar09** [comment]
> `[WARNING] <file>:[<line>] (indentation) Indentation: 'lambd...`

@joca-bt which line?
Please recheck the issue we have tried to fix lambda with #8719

**Comment by romani** [comment]
issue is not following required template, it is hard to understand it.
